Update GPU fleet page metrics tables and screenshots (#36548)
DataDog/documentation: content/en/gpu_monitoring/fleet.md

> The [GPU Fleet page][0] provides a detailed inventory of all of your GPU-accelerated hosts for a specified time frame. Use this view to uncover inefficiencies through resource telemetry, ranging from performance and usage metrics to costs. This page also surfaces Datadog's OOTB provisioning and performance optimization recommendations for your devices, to help you maximize the value of your GPU spend. 

Reviewer: "OOTB" is used here and on lines 40 and 58 but is never spelled out. Consider replacing with "built-in" or spelling out "out-of-the-box" on first use. (Pre-existing, not introduced by this PR.)

Author: 👍 Replaced all 3 occurrences of OOTB with "built-in" — the mention on line 12 plus the "OOTB recommendations:" headers further down the page.

> ## Breakdown your fleet by any tag

Reviewer: "Breakdown" is a noun; the verb form is two words: "break down."

```suggestion
## Break down your fleet by any tag
```

> Use quick filter dropdowns at the top of the page to filter by a specific **Provider**, **Device Type**, **Cluster**, **Region**, **Service**, **Data Center**, **Environment** or **Team**.

Reviewer: Oxford comma missing before "or."

```suggestion
Use quick filter dropdowns at the top of the page to filter by a specific **Provider**, **Device Type**, **Cluster**, **Region**, **Service**, **Data Center**, **Environment**, or **Team**.
```

> You can also **Search** or **Group** by other tags in the fields shown below. For example, you can select the toggle for Host and then group by `Team` to view a table entry for each unique team. Click the **`>`** button next to any entry to see the hosts used by that team and the GPU devices accelerating those hosts.

Reviewer: "The fields shown below" is directional language that breaks when content is reordered or consumed by AI. After the restructure, the filters image is no longer immediately below this text. Consider something like: "You can also **Search** or **Group** by other tags using the search and group-by fields."

Author: 👍 Replaced "fields shown below" with "search and group-by fields" so the wording isn't tied to image position.

**Note**: You can only **Group by** one additional tag.

If you select _Cluster_ or _Host_, you can click on the **`>`** button next to each table entry to view a cluster's hosts or a host's devices, respectively. 

> 

Reviewer: Alt text on the updated screenshots could be more descriptive. For example, this image uses "A host entry in the table" — consider mentioning what's visible (service groupings, device types, the expand button). Same for `filters_and_groupings-3.png`: "The menu for filtering and grouping in the GPU Fleet page" could mention the specific filter dropdowns shown.

Author: 👍 Rewrote alt text on both updated screenshots:
- `host_row_expansion`: "A host entry in the table" → "GPU Fleet table showing services with their device types, with the row expand button highlighted"
- `filters_and_groupings`: "The menu for filtering and grouping in the GPU Fleet page" → "Filter dropdowns and Group by selector at the top of the GPU Fleet page"

| Metric                   | Definition                                                                                                                                                                                                              | Metric Name                                        |
| ------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------- |
| Provisioned Devices      | Breakdown of provisioned devices by active and effective devices.                                                                                                                                                       | `gpu.device.total`                                 |
| Allocated Devices        | (only available if using Kubernetes) Count of devices that have been allocated to a workload.                                                                                                                           | `gpu.device.total`                                 |

> | Active Devices           | Count of devices that are actively used for a workload / busy. If using Kubernetes: count of allocated devices that are actively used for a workload.                                                                   | `gpu.gr_engine_active`                             |

Reviewer: Inconsistent capitalization in parenthetical qualifiers across the tables. Some start lowercase — `(only available if using Kubernetes)` — while others start uppercase — `(Only available if System Probe enabled)`. Recommend capitalizing the first word after the opening paren consistently throughout both tables.

Author: 👍 Standardized parenthetical qualifiers to uppercase throughout both tables. `(Only available if using Kubernetes)` now matches `(Only available if System Probe enabled)` and the other uppercase parentheticals.

> | Effective Devices        | Count of devices that are used and working for more than 50% of their lifespan.                                                                                                                                         | `gpu.sm_active`                                    |

Reviewer: Style guide recommends not using `/` to mean "or." Same applies to the matching row in the inventory table (line 142).

```suggestion
| Active Devices           | Count of devices that are actively used for a workload or are busy. If using Kubernetes: count of allocated devices that are actively used for a workload.                                                                   | `gpu.gr_engine_active`                             |
```

> | Core Utilization         | (Only available if System Probe enabled) `Cores Used/Cores Limit` for GPU processes. Measure of Temporal Core Utilization.                                                                                              | `gpu_core_utilization`                             |

Reviewer: "More than 50% of their lifespan" — is "lifespan" referring to the device's hardware lifespan, or the measurement time interval? This could be confusing for readers. Consider clarifying, for example: "...more than 50% of the selected time frame."

Author: 👍 Changed "more than 50% of their lifespan" to "more than 50% of the selected time frame" in both tables so it's clear it refers to the queried time frame, not device hardware lifetime.

| PCIe RX Throughput       | Bytes received through PCI from the GPU device per second.                                                                                                                                                              | `gpu.pci.throughput.rx`                            |
| PCIe TX Throughput       | Bytes transmitted through PCI to the GPU device per second.                                                                                                                                                             | `gpu.pci.throughput.tx`                            |
| Graphics Engine Activity | Fraction of time the GPU was performing any compute work during the interval. A coarse signal of whether the GPU is busy or idle.                                                                                       | `gpu.gr_engine_active`                             |
| GPU Utilization          | Average % of time each streaming multiprocessor was active (lower values indicate idle time).                                                                                                                           | `gpu.sm_active`                                    |

> | Power                    | Power usage for the GPU device.<br>**Note**: On GA100 and previous architectures, this represents the instantaneous power at that moment.<br>For newer architectures, it represents the average power draw (Watts) over one second. | `gpu.power.usage`                                  |

Reviewer: Minor inconsistency: some definitions in this table use `%` ("Average % of time", "Ratio (%) of GPU memory") while others spell out "Percentage" ("Percentage of time the CPU spent"). Either form works, but pick one for consistency within the table.

Author: 👍 Standardized on `%` everywhere in both tables. CPU Utilization, GPU Idle, and the other percentage-based definitions now consistently use `%` instead of "Percentage" or "percent".

| CPU Utilization          | Percentage of time the CPU spent running user space processes.                                                                                                                                       | `system.cpu.user`                                  |
| Host Uptime              | Time since the host was last started.                                                                                                                                                                                   | `system.uptime`                                    |
| Host I/O Utilization      | % of CPU time during which I/O requests were issued to the GPU device.                                                                                                                                                  | `system.io.util`                                   |
| Host Memory              | % of usable memory in use.                                                                                                                                                                                    | `system.mem.pct_usable`                            |
{{% /collapse-content %}} 

> If you've selected an additional tag to group by---for example, _team_---every unique timeseries in the Summary Graph corresponds to a team's value for the selected metric.

Reviewer: Triple hyphens render as em dashes but aren't proper Markdown em dashes. Use `—` (em dash) with no spaces.

```suggestion
If you've selected an additional tag to group by—for example, _team_—every unique timeseries in the Summary Graph corresponds to a team's value for the selected metric.
```
